src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -66,6 +66,11 @@ object DockerPlugin extends AutoPlugin {
 
   override def projectConfigurations: Seq[Configuration] = Seq(Docker)
 
+  override lazy val globalSettings: Seq[Setting[_]] = Seq(
```

Comment:
I wonder if we should start using `globalSettings` at this point. I would favor a separate PR that moves the `projectSettings` defaults into the `globalSettings`. 

Or we add a small comment that we are now starting [to follow sbt best practices as documented](https://www.scala-sbt.org/release/docs/Plugins-Best-Practices.html#Provide+default+values+in)

Reply:
I figured we can start at least for new ones with pure values. I'll add the comment there.